src/masternodes/mn_rpc.cpp

```diff
@@ -3032,10 +3040,26 @@ UniValue listaccounthistory(const JSONRPCRequest& request) {
         if (!optionsObj["token"].isNull()) {
             tokenFilter = optionsObj["token"].get_str();
         }
+
+        if (!optionsObj["txtype"].isNull()) {
+            const auto str = optionsObj["txtype"].get_str();
+            if (str.size() == 1) {
+                txType = CustomTxCodeToType(str[0]);
+            }
+        }
+        if (!optionsObj["limit"].isNull()) {
+            limit = (uint32_t) optionsObj["limit"].get_int64();
+            depth = std::numeric_limits<decltype(depth)>::max();
+        }
+        if (limit == 0) {
+            limit = std::numeric_limits<decltype(limit)>::max();
```

Comment:
If the `limint` is max of uint32_t, will the API call timeout?

Reply:
I don't get what timeout you mean?